### PR TITLE
feat: Try running Prettier on exit

### PR DESCRIPTION
**Pull request by onurtemizkan**
Resolves: https://github.com/getsentry/sentry-wizard/issues/521

### CODE REVIEW

**Comment by Lms24** [comment]
Oof I totally missed this PR, Sorry! I'll take a look soon!

File: src/run.ts [hunk 1]
```diff
@@ -127,6 +127,8 @@ export async function run(argv: Args) {
     default:
       clack.log.error(`No setup wizard selected!`);
   }
+
+  await runPrettierIfInstalled();
```
Comment (Lms24) [inline src/run.ts:131]: h: let's move this call inside the individual run*wizard functions. If we leave it here, the prompt would be rendered after the outro messages which looks a bit off.

Comment (onurtemizkan) [reply src/run.ts]: @Lms24, moved the prettier runs inside `nextjs`, `remix`, `svelte` and `react-native` wizards.

**Review by Lms24 — APPROVED** [review]
Thanks Onur! Tested with SvelteKit and it worked well. I think there's a limitation we might need to address depending on user feedback which is that we might need to limit applying prettier to the changed files. Should probably be doable since we know which files we generally modify. We could also use git commands to find new/changed files. 

I'll merge this in for now but feel free to work on this if you have free cycles.

**Pull request merged**